Cargo.toml

```diff
@@ -94,6 +94,7 @@ wait-timeout = "0.2"
 walkdir = { workspace = true, optional = true }
 xz2 = "0.1.3"
 zstd = "0.13"
+itertools = "0.12.0"
```

Comment:
Nit: please keep this in alphabetic order. In keeping with the style for other dependencies, prefer eliding trailing zero version components.

Reply:
Done.